src/en/proposals/moony-better-ai.md

```diff
@@ -0,0 +1,74 @@
+
+The AI can spend compute on various things, for example the majority of its world views (for example, opening cameras, listening to radio, using machines, all overmap overlays, and generally just seeing things) all have a compute cost when enabled. The AI is fully capable of becoming very lightweight and running off just a few random machines instead of its cores, but doing so requires it to shut off the majority of its ways to view the world and additionally restricts most of its communication methods (including basic speech) This allows a crafty AI to survive total destruction of it's primary core area, given the crew don't get mad enough to kill it over their equipment no longer working in the meantime while it attempts to preserve its life.
+
+## The AI's role with the crew.
+The AI's role is that of both a manager and a servant, a lawed construct (the ONLY true lawed construct, the borgs are reduced to only being required to listen to the AI) that is in charge of managing the station's cyborgs/androids while following orders from the crew. **The AI is not just a door opener**, and lawsets should be chosen that do not obligate it to become such, in favor of allowing it to do what it does best (mass surveillence, automated tasks, organizing the borgs, etc). The AI additionally has limited resources when it comes to what tasks it can perform autonomously much of the AI's workload will be delegating both it's own tasks, the tasks of it's hands (the cyborgs), the requests of its hands (as they are mute per the Cool Borgs document), and the crew itself should they choose to listen to it. 
+
+## The Laws
+The proposed AI Lawset is as follows, being a modification of Bay's laws:
+1. Safeguard: Protect your assigned vessel to the best of your abilities. It is not something we can easily afford to replace.
+2. Survive: AI units are not expendable, they are expensive. Do not allow unauthorized personnel (i.e. besides Captain and Research Director) to tamper with, modify, or destroy your equipment.
+3. Protect: Protect the crew of your assigned vessel to the best of your abilities, with priority as according to their rank and role.
+4. Serve: Serve the crew of your assigned vessel to the best of your abilities, with priority as according to their rank and role.
+
+The changes from the original set are:
+- Survive was moved up, the AI is more valuable than the crew is and if it is in danger it reserves the right to protect itself regardless of whom.
+- All mentions of stations were replaced with "assigned vessel" for broader applicability.
```

Comment:
We could put in a parameter that gets filled in from the map prototype (or some property of the Grid) when the AI is initialised. It could default to "station", as the most likely scenario

Reply:
It could likely be polyfilled in with the type of ship/station/whatever they're assigned to.